### PR TITLE
Add support for Popularity tag

### PR DESCRIPTION
**Pull request by larchunix**
Also bump the version of the AUR RPC API to v4.

### CODE REVIEW

File: src/aur.c [hunk 1]
```diff
@@ -385,6 +404,28 @@ static int json_integer (void * ctx, long long val)
 	return 1;
 }
 
+static int json_double (void * ctx, double val)
```
Comment (f2404) [inline src/aur.c]: Looks like AUR_POPULARITY is worth adding into json_string() as well.

File: src/aur.c [hunk 2]
```diff
@@ -282,6 +294,13 @@ const char *aur_pkg_get_maintainer (const aurpkg_t *pkg)
 	return NULL;
 }
 
+double aur_pkg_get_popularity (const aurpkg_t *pkg)
+{
+	if (pkg != NULL)
+		return pkg->popularity;
+	return 0;
```
Comment (f2404) [inline src/aur.c]: Maybe return 0.0 to be clear?

File: src/aur.c (hunk 1 above)
Comment (larchunix) [inline src/aur.c]: I do not understand why this is needed. Could you clarify that for me ?

Comment (f2404) [inline src/aur.c]: I cannot tell off the top of my head - need to investigate that.
But all other tags are there...

Comment (f2404) [inline src/aur.c]: Yes, only the tags of string type should be there.
I will have to clean this function up from unneeded tags.

Comment (larchunix) [inline src/aur.c]: I agree, I removed non string attributes (larchunix/package-query@fa823c2) and didn't see any regression.

Comment (f2404) [inline src/aur.c]: I've opened a PR for that: archlinuxfr/package-query#61

**Comment by f2404** [comment]
Looks good to me.

File: src/aur.c (hunk 1 above)
Comment (larchunix) [inline src/aur.c]: Perfect (your changes are smarter than mine btw).

Comment (f2404) [inline src/aur.c]: Thanks :)

**Comment by Skunnyk** [comment]
Yai, good idea to add the popularity flag :)
Works for me©

**Comment by f2404** [comment]
@larchunix 
Can you please resolve the conflicts?

**Comment by larchunix** [comment]
Rebased.

**Comment by f2404** [comment]
Thanks!

**Pull request merged**